src/encoder.pre

```diff
@@ -0,0 +1,24 @@
+/* Copyright (C) 2022 Simo Sorce <simo@redhat.com>
```

Comment:
```suggestion
/* Copyright (C) 2022 - 2023 Simo Sorce <simo@redhat.com>
```

Reply:
Heh .... In this case I just copied over code written in 2022 :-)